generators/entity-server/files-couchbase.js

```diff
@@ -21,12 +21,23 @@ const { SERVER_MAIN_SRC_DIR, SERVER_MAIN_RES_DIR } = require('../generator-const
 const entityCouchbaseFiles = {
   dbChangelog: [
     {
-      condition: generator => generator.searchEngineCouchbase && !generator.skipDbChangelog,
+      condition: generator => !generator.skipDbChangelog && !generator.embedded,
+      path: SERVER_MAIN_RES_DIR,
+      templates: [
+        {
+          file: 'config/couchmove/changelog/entity.n1ql',
+          renameTo: generator => `config/couchmove/changelog/V${generator.changelogDate}__${generator.entityInstance.toLowerCase()}.n1ql`,
+        },
+      ],
+    },
+    {
+      condition: generator => generator.searchEngineCouchbase && !generator.skipDbChangelog && !generator.embedded,
       path: SERVER_MAIN_RES_DIR,
       templates: [
         {
           file: 'config/couchmove/changelog/entity.fts',
-          renameTo: generator => `config/couchmove/changelog/V${generator.changelogDate}__${generator.entityInstance.toLowerCase()}.fts`,
+          renameTo: generator =>
+            `config/couchmove/changelog/V${parseInt(generator.changelogDate, 10) + 10}__${generator.entityInstance.toLowerCase()}.fts`,
```

Comment:
Changelogs should have different versions, and since we can have both n1ql and fts changelong for an entity, we need to increment it a bit. I tried first with 1 but it doesn't work because a lot of entities in the sample have just 1s of difference. So I incremented by 10.

If you have a better idea on how to handle this I'm all ears.

Reply:
This is not the best place, but since this is couchbase specific, and we are not modular yet, it's ok.